product_docs/docs/efm/4/07_using_efm_utility.mdx

```diff
@@ -16,7 +16,7 @@ Failover Manager provides the `efm` utility to assist with cluster management. T
 
 <div id="efm_allow_node" class="registered_link"></div>
 
-`efm allow-node <cluster_name>`
+`efm allow-node <cluster_name> <address>`
 
 Invoke the `efm allow-node` command to allow the specified node to join the cluster. When invoking the command, provide the name of the cluster and the IP address of the joining node.
```

Comment:
While we're in here, can you remove "IP" also? The word "address" is sufficient, and it could be either an IP or a host name. No big deal though.